File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication69
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            Control c = new DevExpress.XtraTreeList.Demos.Tutorials.NodeChecking();
            c.Dock = DockStyle.Fill;
            this.Controls.Add(c);
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication69
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			Dim c As Control = New DevExpress.XtraTreeList.Demos.Tutorials.NodeChecking()
			c.Dock = DockStyle.Fill
			Me.Controls.Add(c)
		End Sub
	End Class
End Namespace